server/utils/nunjucksSetup.ts

```diff
@@ -89,5 +89,9 @@ export default function nunjucksSetup(app: express.Express, applicationInfo: App
 
   njkEnv.addFilter('pluralise', pluralise)
 
+  njkEnv.addFilter('formatTelephone', (telephoneNumber: string) => {
+    return telephoneNumber === 'UNKNOWN' || telephoneNumber === undefined ? 'No phone number provided' : telephoneNumber
+  })
+
```

Comment:
I don't think we need this as we shouldn't have to handle numbers with `UNKNOWN` - we'll only be showing visits that originated in this service.

So, suggest just putting in the template file:
```
...  mainContactNumber | d("No phone number provided")   ...
```